static/src/javascripts/projects/common/modules/commercial/article-body-adverts.js

```diff
@@ -75,7 +75,7 @@ define([
 
                 ads.push($ad);
                 return new Promise(function (resolve) {
-                    fastdom.write(function () {
+                    idleFastdom.write(function () {
```

Comment:
Just a heads up - I've actually been looking at this very write today. It seems the way we're finding spaces and writing elements asynchronously can create some race conditions. That means we might have to either do some reads and writes synchronously, or do some careful scheduling that prevents ads being injected into spaces that are no longer empty. 

If you're doing some performance investigation, though, I'm wary of doing anything that might add another variable to your testing.